lib/jar.ts

```diff
@@ -138,11 +141,13 @@ async function getMavenPackageInfo(sha1: string): Promise<MavenPackageInfo> {
             ),
           );
         }
-
         if (res.response.docs.length > 1) {
-          debug('Got multiple results for sha1, only returning first one');
+          const sha1Target = path.parse(targetPath).base;
+          debug('Got multiple results for sha1, looking for', sha1Target);
+          res.response.docs.map((doc) =>
```

Comment:
You don't need `map` for this. You can just go with `forEach`. `map will create a new array`, while `forEach` will just loop until you find smth.

Reply:
UPD: Actually `Array.find` suites here in the best way